docs/examples_gcp.rst

```diff
@@ -128,8 +128,8 @@ No AWS keys required, but all files being copied must have a status of released.
 
 ::
 
-  import encode_utils.transfer_to_gcp as gcp 
-  import encode_utils.connection as euc
+  import igvf_utils.transfer_to_gcp as gcp 
+  import igvf_utils.connection as euc
   conn = euc.Connection("prod")
```

Comment:
again there is distinction between prod and dev for ENCODE that is not necessarily there for IGVF, and I don't know where the URLs for the dev/prod are set.